Venmo/src/main/java/com/braintreepayments/api/VenmoAccountNonce.java

```diff
@@ -30,13 +30,17 @@ public class VenmoAccountNonce extends PaymentMethodNonce {
     private static final String VENMO_LAST_NAME_KEY = "lastName";
     private static final String VENMO_PHONE_NUMBER_KEY = "phoneNumber";
     private static final String VENMO_PAYMENT_METHOD_USERNAME_KEY = "userName";
+    private static final String VENMO_BILLING_ADDRESS_KEY = "billingAddress";
+    private static final String VENMO_SHIPPING_ADDRESS_KEY = "shippingAddress";
 
     private String email;
     private String externalId;
     private String firstName;
     private String lastName;
     private String phoneNumber;
     private String username;
+    private PostalAddress billingAddress;
+    private PostalAddress shippingAddress;
```

Comment:
We should add these methods to the `Parcelable` implementation. As of now, if this data is parceled `billingAddress` and `shippingAddress` will be `null` after deserialization.